packages/imperative/src/config/src/ProfileInfo.ts

```diff
@@ -15,13 +15,15 @@ import * as path from "path";
 import * as url from "url";
 import * as jsonfile from "jsonfile";
```

Comment:
I can't remember where I saw it being used, but noticed it was a dependency. I've been using it mostly for the `spaces` option to fix the output formatting, otherwise the object is written to disk without any new lines.

But, if `fs.writeFileSync` is efficient enough to do this behind the scenes, I definitely would prefer using that instead 😋 I'll try it out and give an update once confirmed 👍

Reply:
I looked into it a bit, and we were using `jsonfile` before these changes, so I will keep it as-is for now. I also noticed that we are using it in other places within imperative, so maybe these adjustments could be done as a separate enhancement 🙂